sdcm/cluster.py

```diff
@@ -1713,6 +1714,7 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
 
         if alternator_port:
             scylla_yml['alternator_port'] = alternator_port
+            scylla_yml['alternator_write_isolation'] = WriteIsolation.ALWAYS_USE_LWT.value
```

Comment:
why not using the parameter from sct_config.py ?

Reply:
agreed to leave as is since it only sets a default value.